src/lib.rs

```diff
@@ -385,18 +396,28 @@ where
     A: Annotation<KvPair<K, V>>,
     A::Archived: for<'any> CheckBytes<DefaultValidator<'any>>,
     I: Archive + for<'any> CheckBytes<DefaultValidator<'any>>,
+    <K as Archive>::Archived: Hash,
```

Comment:
I think this is hash bound is deprecated now.

Reply:
indeed